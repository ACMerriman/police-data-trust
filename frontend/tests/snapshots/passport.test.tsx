import * as React from "react"
import renderer from "react-test-renderer"
import Passport from "../../pages/apply/index"

it("renders correctly", () => {
  const tree = renderer.create(<Passport />).toJSON()
  expect(tree).toMatchSnapshot()
})
