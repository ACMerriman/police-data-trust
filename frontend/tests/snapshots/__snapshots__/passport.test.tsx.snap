// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <section
    className="enrollmentSection"
  >
    <div
      className="header"
    >
      <img
        alt="A blue circular logo design reads 'National Police Data Coalition'"
        className="logo medium"
        src="/NPDCLogo.svg"
      />
      <h1>
        Passport Account Application
      </h1>
    </div>
    <p
      className="passportIntro"
    >
      Hello 
      <em>
        Herbert Placeholder
      </em>
      , thank you for your continued interest in the National Police Data Coalition.
      <br />
      <br />
      We are able to provide access to legally protected data to users with the appropriate permissions. This form will submit your profile for approval.
    </p>
    <form
      className="passportForm"
    >
      <fieldset>
        <div
          className="defaultInputContainer inputContainer false"
        >
          <label
            htmlFor="streetAddressInput"
          >
            Street Address:
          </label>
          <input
            aria-describedby="streetAddressError"
            aria-required="true"
            className="inputField large"
            id="streetAddressInput"
            max={null}
            min={null}
            name="streetAddress"
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
        <div
          className="defaultInputContainer inputContainer false"
        >
          <label
            htmlFor="cityOrTownInput"
          >
            City Or Town:
          </label>
          <input
            aria-describedby="cityOrTownError"
            aria-required="true"
            className="inputField undefined"
            id="cityOrTownInput"
            max={null}
            min={null}
            name="cityOrTown"
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
        <div
          className="defaultInputContainer false"
        >
          <label
            htmlFor="stateSelectInput"
          >
            State:
          </label>
          <input
            aria-required="true"
            className="stateSelect false"
            id="stateSelectInput"
            list="stateSelectList"
            name="stateSelectInput"
            onChange={[Function]}
            type="text"
          />
          <datalist
            id="stateSelectList"
          >
            <option
              value="AK"
            >
              Alaska
            </option>
            <option
              value="AL"
            >
              Alabama
            </option>
            <option
              value="AR"
            >
              Arkansas
            </option>
            <option
              value="AS"
            >
              American Samoa
            </option>
            <option
              value="AZ"
            >
              Arizona
            </option>
            <option
              value="CA"
            >
              California
            </option>
            <option
              value="CO"
            >
              Colorado
            </option>
            <option
              value="CT"
            >
              Connecticut
            </option>
            <option
              value="DC"
            >
              District of Columbia
            </option>
            <option
              value="DE"
            >
              Delaware
            </option>
            <option
              value="FL"
            >
              Florida
            </option>
            <option
              value="GA"
            >
              Georgia
            </option>
            <option
              value="GU"
            >
              Guam
            </option>
            <option
              value="HI"
            >
              Hawaii
            </option>
            <option
              value="IA"
            >
              Iowa
            </option>
            <option
              value="ID"
            >
              Idaho
            </option>
            <option
              value="IL"
            >
              Illinois
            </option>
            <option
              value="IN"
            >
              Indiana
            </option>
            <option
              value="KS"
            >
              Kansas
            </option>
            <option
              value="KY"
            >
              Kentucky
            </option>
            <option
              value="LA"
            >
              Louisiana
            </option>
            <option
              value="MA"
            >
              Massachusetts
            </option>
            <option
              value="MD"
            >
              Maryland
            </option>
            <option
              value="ME"
            >
              Maine
            </option>
            <option
              value="MI"
            >
              Michigan
            </option>
            <option
              value="MN"
            >
              Minnesota
            </option>
            <option
              value="MO"
            >
              Missouri
            </option>
            <option
              value="MS"
            >
              Mississippi
            </option>
            <option
              value="MT"
            >
              Montana
            </option>
            <option
              value="NC"
            >
              North Carolina
            </option>
            <option
              value="NE"
            >
              Nebraska
            </option>
            <option
              value="NH"
            >
              New Hampshire
            </option>
            <option
              value="NJ"
            >
              New Jersey
            </option>
            <option
              value="NM"
            >
              New Mexico
            </option>
            <option
              value="NV"
            >
              Nevada
            </option>
            <option
              value="NY"
            >
              New York
            </option>
            <option
              value="ND"
            >
              North Dakota
            </option>
            <option
              value="OH"
            >
              Ohio
            </option>
            <option
              value="OK"
            >
              Oklahoma
            </option>
            <option
              value="OR"
            >
              Oregon
            </option>
            <option
              value="PA"
            >
              Pennsylvania
            </option>
            <option
              value="PR"
            >
              Puerto Rico
            </option>
            <option
              value="RI"
            >
              Rhode Island
            </option>
            <option
              value="SC"
            >
              South Carolina
            </option>
            <option
              value="SD"
            >
              South Dakota
            </option>
            <option
              value="TN"
            >
              Tennessee
            </option>
            <option
              value="TX"
            >
              Texas
            </option>
            <option
              value="UT"
            >
              Utah
            </option>
            <option
              value="VA"
            >
              Virginia
            </option>
            <option
              value="VT"
            >
              Vermont
            </option>
            <option
              value="WA"
            >
              Washington
            </option>
            <option
              value="WI"
            >
              Wisconsin
            </option>
            <option
              value="WV"
            >
              West Virginia
            </option>
            <option
              value="WY"
            >
              Wyoming
            </option>
          </datalist>
        </div>
        <div
          className="defaultInputContainer inputContainer false"
        >
          <label
            htmlFor="zipCodeInput"
          >
            Zip Code:
          </label>
          <input
            aria-describedby="zipCodeError"
            aria-required="true"
            className="inputField small"
            id="zipCodeInput"
            max={99999}
            min={0}
            name="zipCode"
            onChange={[Function]}
            type="number"
            value=""
          />
        </div>
      </fieldset>
      <div
        className="defaultInputContainer false"
      >
        <label
          htmlFor="responseTextArea"
        >
          Why are you signing up to the NPDC?:
        </label>
        <textarea
          aria-describedby="responseCounter null"
          aria-required="true"
          cols={52}
          id="responseTextArea"
          maxLength={500}
          onChange={[Function]}
          rows={7}
        />
        <div
          className="responseSubtext"
        >
          <p
            id="responseCounter"
          >
            0
            /
            500
          </p>
        </div>
      </div>
      <button
        className="primaryButton"
        onClick={[Function]}
        type="submit"
      >
        Submit
      </button>
    </form>
    <div
      className="callToActionContainer"
    >
      <p>
        Is the publically available data sufficient for your needs?
      </p>
      <a
        className="callToActionLink"
        href="/dashboard"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Return to dashboard
      </a>
    </div>
  </section>
</div>
`;
